libgraph/include/katana/Properties.h

```diff
@@ -573,10 +571,8 @@ struct StructProperty
 
     auto type = res.ValueOrDie();
     arrow::FixedSizeBinaryBuilder builder(type);
-    // TODO(lhc): replace this with AppendEmptyValues() on Arrow >= 3.0.
-    katana::PODVector<uint8_t> data(sizeof(T) * num_rows);
-
-    if (auto res = builder.AppendValues(data.data(), num_rows); !res.ok()) {
+    builder.ReserveData(num_rows);
+    if (auto res = builder.AppendEmptyValues(num_rows); !res.ok()) {
```

Comment:
ditto

Reply:
Thank you for letting this know. Is `Reserve()` the right function for this?